x-pack/plugins/apm/public/components/shared/Summary/index.tsx

```diff
@@ -11,7 +11,7 @@ import { euiStyled } from '../../../../../../../src/plugins/kibana_react/common'
 import { Maybe } from '../../../../typings/common';
 
 interface Props {
-  items: Array<Maybe<React.ReactElement>>;
+  items: Array<Maybe<React.ReactElement | string>>;
```

Comment:
Not sure we should change this component which is used in other places too as part of this PR. Can you try to adapt the props we pass in? E.g. change `'text'` to `<>{text}</>`.

Reply:
Updated here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6